app/views/components/_navbar.html.erb

```diff
@@ -1,7 +1,13 @@
-<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
-  <%= link_to "#", class: "navbar-brand" do %>
-    <%= image_tag "https://res.cloudinary.com/xanldn89/image/upload/v1629881217/_logo_pd7mff.png", class:"navbar-logo" %>
-    <% end %>
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon" style='height: 64px;'>
```

Comment:
Actually already added height in the navbar scss on my end already so no need for inline style 😅 

```suggestion
<div class="navbar navbar-expand-sm navbar-light navbar-lewagon" >
```